### ISSUE TITLE
PD 3.1 EPR support

### ISSUE THREAD

**Issue by drom** [wagiminator/CH32X035-USB-PD-Tester#1]
Great project!
Do you plan supporting USB PD 3.1 EPR profiles {28, 36, 48}V ?
There are several 140W = 28V * 5A power bricks on the market right now to test.

**Comment by wagiminator** [comment]
Thank you. I don't plan to do this at the moment. The hardware would have to be adapted so that it can tolerate the higher voltages (in particular, the cheap LDO should be replaced by a buck converter). When I have the appropriate power bricks at hand at some point, I'll definitely make a project out of them.

**Comment by andelf** [comment]
I made something similar. :)

https://github.com/ch32-rs/ch32x0-hal/issues/1

**Comment by wagiminator** [comment]
Coooooool! Now I know why I didn't get hardware-I2C to work....

**Comment by andelf** [comment]
Yes. A sad story. It is documented in the datasheet, with only a single line of description.😤

**Comment by wagiminator** [comment]
Ah, I just found the sentence in the datasheet. If I hadn't read it over, I could have saved myself frustrating hours with the logic analyzer.
